pkg/adaptors/oidcclient/oidcclient.go

```diff
@@ -92,6 +93,11 @@ func (c *client) GetTokenByAuthCode(ctx context.Context, in GetTokenByAuthCodeIn
 		LocalServerBindAddress: in.BindAddress,
 		LocalServerReadyChan:   localServerReadyChan,
 	}
+
+	for key, value := range c.extraURLParams {
+		config.AuthCodeOptions = append(config.AuthCodeOptions, oauth2.SetAuthURLParam(key, value))
```

Comment:
This seems add extra params to the authentication request. Is it okay?

Which one do you add extra params to?

- Authentication Request (https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest)
- Token Request (https://openid.net/specs/openid-connect-core-1_0.html#TokenRequest)

Reply:
In our case we want to add it to the authentication request.

Now that you mention it I can see that the commit message is wrong, but the code does what we want it to do. Let me know if you would like me to make changes.

Thanks for reviewing.

Reply:
I got it.